src/store/balances/balances-state.ts

```diff
@@ -0,0 +1,11 @@
+import {BalanceObject} from '../../interfaces/balance-object.interface';
+import {LoadableEntityState} from '../types';
+import {createEntity} from '../utils/create-entity';
+
+export interface BalancesState {
```

Comment:
let's call this `WalletState` as data here will be related to connected wallet.

Reply:
and move `pendingSwapTransaction` from `SwapRoutesState` into this new `WalletState`. this will be more logical